dags/common/bucket.py

```diff
@@ -0,0 +1,8 @@
+from airflow.models import Variable
+
+
+def bucket_connection():
+    hostbucket = Variable.get("HOSTBUCKET_C2")
+    passwordbucket = Variable.get("PASSWORDBUCKET_C2")
+    userbucket = Variable.get("USERBUCKET_C2")
```

Comment:
_Rindfleischetikettierungsüberwachungsaufgabenübertragungsgesetz_

```suggestion
    host = Variable.get("HOST_BUCKET_C2")
    password = Variable.get("PASSWORD_BUCKET_C2")
    user = Variable.get("USER_BUCKET_C2")
```
Pour que ça soit un poil plus sympa à lire et qu'on ai pas l'impression de se faire marteler la tête à coups de "bucket-bucket-bucket" :P.
Je remplacerais aussi `BUCKET` par `FTP` pour être le plus explicite quand on tombe dessus sans contexte.